pitop/camera/camera.py

```diff
@@ -32,10 +32,11 @@ class Camera(Stateful, Recreatable):
 
     def __init__(self,
                  index=None,
-                 resolution=None,
+                 resolution=(640, 480),
```

Comment:
Why? This is an undocumented change

Reply:
So a user can just do `cam = Camera()` and not have to worry about resolution - especially useful for beginners. It's also good for us to have a standard resolution we develop algorithms for, so at least we know the main use-case

Reply:
Let's drop this from here and implement a better solution in this Issue: https://github.com/pi-top/pi-top-Python-SDK/issues/326